app/views/evaluations/_phases_table.html.erb

```diff
@@ -14,26 +13,24 @@
         <th data-label="Challenge Title" scope="row" class="text-top">
           <%= challenge_phase_title(phase.challenge, phase) %>
         </th>
-        <td data-label="Assigned to me" class="text-top">
-          <%= assigned_submissions_count(current_user, phase.challenge, phase) %>
-        </td>
-        <td data-label="Remaining to evaluate" class="text-top">
-          <%= remaining_evaluations_count(current_user, phase.challenge, phase) %>
+        <td data-label="# of Submissions to Evaluate" class="text-top">
+          <%= link_to submissions_evaluation_path(phase) do %>
+            <%= assigned_submissions_count(current_user, phase.challenge, phase) %> of 
+            <%= remaining_evaluations_count(current_user, phase.challenge, phase) %>
+          <% end %>
         </td>
-        <td data-label="Due by" class="text-top">
+        <td data-label="Evaluation Due Date" class="text-top">
           <% if phase.evaluation_form %>
-            <%= phase.evaluation_form.closing_date %>
+            <%= phase.evaluation_form.closing_date.strftime('%m/%d/%Y') %>
           <% else %>
             N/A
           <% end %>
         </td>
-        <td>
-          <div class="display-flex flex-column grid-row grid-gap-1 padding-bottom-2">
-            <%= link_to submissions_evaluation_path(phase), class: "usa-button font-body-2xs text-no-wrap width-full" do %>
-              View Submissions
-            <% end %>
-          </div>
-        </td>  
+        <td data-label="Evaluation Status" class="text-top">
+          <span class="text-top text-no-wrap usa-tag <%= evaluation_submission_assignment_status_color(phase) %>">
+            <%= phase.evaluation_status.to_s.titleize %>
```

Comment:
![Screenshot 2025-02-24 at 9 35 27 AM](https://github.com/user-attachments/assets/feb92d88-b1fd-4dbc-bd62-878aa1212e61)
![Screenshot 2025-02-24 at 9 38 15 AM](https://github.com/user-attachments/assets/cb91625a-d7e9-4283-86b7-3997f4f57483)

second row of My Evaluations shows "In Progress" status, however my actual status for this phase is "Not Started" because my only assignment is not started yet. this Evaluation Status is only referring to the evaluator's view of their progress, not for the overall Phase.